server/src/lib/model/db.ts

```diff
@@ -130,4 +130,22 @@ export default class DB {
   endConnection(): void {
     this.mysql.endConnection();
   }
+
+  async findClipsWithFewVotes(limit: number): Promise<DBClipWithVoters[]> {
+    const [clips] = await this.mysql.exec(
+      `
+      SELECT clips.*, group_concat(votes.client_id) AS voters
+      FROM clips
+        LEFT JOIN votes ON clips.id = votes.clip_id
+      GROUP BY clips.id
+      ORDER BY COUNT(votes.id)
```

Comment:
At some point if would probably be good to add a WHERE clause that eliminated all validate/invalidated clips from this query. For now, it's not a big deal since we have so many invalidated clips